src/Cohesity.Powershell/Scripts/Clone/Copy-CohesityVMwareVM.ps1

```diff
@@ -0,0 +1,286 @@
+function Copy-CohesityVMwareVM {
+    <#
+        .SYNOPSIS
+        Clones the specified VMware virtual machine.
+        .DESCRIPTION
+        Clones the specified VMware virtual machine. The cmdlet can copy VM from remote cluster as well.
+        .NOTES
+        Published by Cohesity
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Copy-CohesityVMwareVM -TaskName "test-clone-task" -SourceId 883 -TargetViewName "test-vm-datastore" -JobId 49402 -VmNamePrefix "clone-" -DisableNetwork -PoweredOn -ResourcePoolId 893
+        Clones the VMware virtual machine with the given source id using the latest run of job id 49402.
```

Comment:
Is this comment or part of the cmdlet ?

Reply:
The cmdlet examples followed by the comments.